pages/pipelines/tags.md

```diff
@@ -0,0 +1,13 @@
+# Pipeline tags
+
+Pipeline tags allow you to tag and search for your pipelines using the search bar. Tags are beneficial when you have many pipelines and would like to group and filter through them quickly.
+
+<%= image "pipeline-tag-search.png", alt: "Imagine showing pipeline tag search" %>
```

Comment:
I should have caught this before sorry, but I just noticed the alt text. I assume there's a typo `Imagine` -> `Image`. In alt text, you don't need to include the "image of" kind of intro, just a short and clear description of what it shows. Perhaps:

```suggestion
<%= image "pipeline-tag-search.png", alt: "The search bar is selected and shows a dropdown with suggested tags." %>
```